data/hai/hai.txt

```diff
@@ -207,6 +213,116 @@ phrase "hai"
 		"Lur"
 		"Sen"
 		"Bin"
+		"Bah"
+		"Ki"
+		"Blep"
+		"Tee"
+		"Say"
+		"Hiya"
+		"Choo"
+		"Gip"
+		"Oo"
+		"Mer"
+		"Ki"
+		"Tou"
+
+phrase "hai first names"
+	word
+		"Tee"
+		"Say"
+		"Hiya"
+		"Choo"
+		"Gip"
+		"Oo"
+		"Mer"
+		"Ki"
+		"Tou"
+		"Cor"
+		"Yo"
+		"Sook"
+		"Woa"
+		"Loh"
+		"Eh"
+		"Ow"
+		"Hey"
+		"Lur"
+		"Sen"
+		"Bin"
+		"Na"
+		"Ya"
+		"Os"
+		"Gem"
+		"Mis"
+		"Ey"
+		"Ra"
+		"Des"
+	word
+		"neep"
+		"ari"
+		"mah"
+		"t'k"
+		"eep"
+		"nem"
+		"oot"
+		"ru"
+		"har"
+		"na"
+		"shi"
+		"en"
+		"ana"
+		"ama"
+		"heep"
+		"kei"
+		"yo"
+	word
+		""
+		""
+		""
+		""
+		""
+		""
+		"aru"
+		"ow"
+		"hey"
+		"lur"
+		"sen"
+		"bin"
+		"chi"
+		"li"
+		"ri"
+
+phrase "hai last names"
+	word
+		"Yeer"
+		"Wah'"
+		"Du'"
+		"Mon"
+		"Luf"
+		"Vai'"
+		"Woh'"
+		"Far"
+		"Yoot"
+		"Imo'"
+	word
+		"tle"
+		"uden"
+		"aru"
+		"ow"
+		"hey"
+		"lur"
+		"sen"
+		"bin"
+		"chi"
+		"li"
+		"ri"
+
+phrase "unfettered chosen names"
+	word
+		"He-"
+		"Si-"
+	word
+		"Yi"
+		"Ra"
```

Comment:
```suggestion
	word
		"He"
		"Si"
	word
		"-"
	word
		"Yi"
		"Ra"
```
Small change but the `-` is intended to separate the two parts, which I think of as Hai abbreviations for some words